components/binding-usage-controller/docs/architecture.md

```diff
@@ -26,7 +26,7 @@ The Binding Usage Controller workflow steps are as follows:
 See the example:
 
   ```
-servicebindingusages.servicecatalog.kyma.cx/tracing-information: '{
+servicebindingusages.servicecatalog.kyma-project.io/tracing-information: '{
   "azure-mysqldb-instance-binding-usage": {
     "injectedLabels": {
       "use-db31ae30-7ecb-11e8-a568-000d3a384f22": "445978"
```

Comment:
In this file, below "examples" there is an image with name which is not changed
<img width="414" alt="screen shot 2018-12-21 at 13 28 41" src="https://user-images.githubusercontent.com/10399074/50342763-b8dd8e80-0524-11e9-9d2a-5ae5919dea04.png">

Reply:
I will change that in #2112. Anyway, good catch. I didn't see that earlier ;)